src/hooks/useActionHandlers/useActionHandlers.ts

```diff
@@ -16,6 +16,7 @@ export function createOnKeyDownHandler<T>(callback?: AnyFunction) {
             callback &&
             [KeyCode.ENTER, KeyCode.SPACEBAR, KeyCode.SPACEBAR_OLD].includes(event.key)
         ) {
+            event.preventDefault();
```

Comment:
Why it is not in callback in Dropdown?

Reply:
It's useful in every use of useActionHandlers, we want only call the callback when key is pressed, not the default behaviour of this key